src/App.tsx

```diff
@@ -1,39 +1,74 @@
+/* eslint-disable no-console */
 import React from 'react';
 import './App.scss';
+import Clock from './components/Clock';
 
 function getRandomName(): string {
   const value = Date.now().toString().slice(-4);
 
   return `Clock-${value}`;
 }
 
-export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
+type Props = {
+  clockName: string;
+  hasClock: boolean;
+  timerID: number;
+};
+
+type State = {
+  clockName: string;
+  timerID: number;
+  hasClock: boolean;
+};
```

Comment:
You have type State and type Props that are actually the same. So you could use only one.